examples/draft-0-10-0/rich/rich.html

```diff
@@ -43,14 +43,13 @@
           this.focus = () => this.refs.editor.focus();
           this.onChange = (editorState) => this.setState({editorState});
 
-          this.handleKeyCommand = (command) => this._handleKeyCommand(command);
+          this.handleKeyCommand = this._handleKeyCommand.bind(this);
```

Comment:
What do you think of `(cmd, state) => this._handleKeyCommand(cmd, state)` instead?